src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java

```diff
@@ -208,6 +209,12 @@ public void onBlockPlace(BlockPlaceEvent placeEvent)
 			for(int i = 0; i < players.size(); i++)
 			{
 				Player otherPlayer = players.get(i);
+
+				// Ignore players in creative or spectator mode to avoid users from checking if someone is spectating near them
+				if(otherPlayer.getGameMode().equals(GameMode.CREATIVE) || otherPlayer.getGameMode().equals(GameMode.SPECTATOR)) {
```

Comment:
Style preference, but for consistency (and the standard convention for Java at least), use `==` for enum comparisons instead of #equals

Reply:
Another option/check to employ may be to use player#canSee.